analogout.py

```diff
@@ -159,17 +159,12 @@ def init(self):
                 # Clear old task
                 if self.task is not None:
                     try:
-                        self.task.close()
-
-                    except DaqError:
-                        # end the task nicely
-                        self.stop()
                         self.close()
-                        msg = '\n AnalogOutput failed to close current task'
-                        raise HardwareError(self, task=self.task, message=msg)
+                    except DaqError: # could extract error code to check if stale reference
```

Comment:
I would like this to test the code for stale reference. Or at least print the error so we don't have another failure mode go silent.